Makefile

```diff
@@ -51,7 +51,7 @@ clean:
 	rm *.o
 
 .PHONY: js
-js: shoco.js
+js: shoco.js	
```

Comment:
I had previously changed the Makefile, adding two new targets. It seems I didn't revert it correctly.